apps/sms/views/shared/js/utils.js

```diff
@@ -940,7 +940,12 @@
         });
 
       return promise.then(
-        () => shimHostIframe.contentWindow.bootstrap(appInstanceId)
+        () => {
+          window.mozSetMessageHandler = function(type, handler) {
```

Comment:
Just first idea that came to my mind, maybe we can think of something smarter - just want to finally enable our tests :)